BE/app/services/persona_service.py

```
@@ -71,73 +74,76 @@ async def generate_persona(
         if oauth_account and oauth_account.access_token:
             access_token = oauth_account.access_token
-            print(f"[Persona] OAuth 토큰 발견, YouTube API로 자막 추출")
+            print(f"[Persona]   └─ OAuth 토큰 발견 → YouTube API로 자막 추출")
 
     # 2. 영상 데이터 조회
-    videos = await _get_channel_videos_with_stats(db, channel_id)
+    print(f"[Persona] 2단계: 영상 데이터 조회...")
+    try:
+        videos = await _get_channel_videos_with_stats(db, channel_id)
+        print(f"[Persona] ✓ 2단계 완료: 영상 {len(videos)}개 조회")
+    except Exception as e:
```

Ruff (0.15.0)

[warning] Do not catch blind exception: `Exception`

(BLE001)